network-layer/Readme.md

```diff
@@ -0,0 +1,31 @@
+### Goal:
+1. Configure the network according to the scheme, using only static routing
+2. Verify the network is working with the ping command (all interfaces from everywhere must ping)
+3. Check the routing tables
+4. Practice using the tracert command
+5. *. Configure loop back interfaces
+
+
+### Solution:
```

Comment:
What are the `pkt` files and why are they here?

Reply:
these are Packet Tracer files. One file need to be fixed, second one is solution.